test/e2e/specs/editor/plugins/image-size.spec.js

```diff
@@ -1,52 +1,60 @@
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+
 /**
  * WordPress dependencies
  */
 const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
 
 test.describe( 'changing image size', () => {
-	test.beforeEach( async ( { requestUtils, pageUtils } ) => {
+	test.beforeAll( async ( { requestUtils } ) => {
 		await requestUtils.activatePlugin( 'gutenberg-test-image-size' );
-		await pageUtils.createNewPost();
+		await requestUtils.deleteAllMedia();
 	} );
 
 	test.afterEach( async ( { requestUtils } ) => {
+		await requestUtils.deleteAllMedia();
+	} );
+
+	test.afterAll( async ( { requestUtils } ) => {
 		await requestUtils.deactivatePlugin( 'gutenberg-test-image-size' );
 	} );
 
 	test( 'should insert and change my image size', async ( {
 		page,
 		pageUtils,
+		requestUtils,
 	} ) => {
-		await pageUtils.insertBlock( { name: 'core/image' } );
-		const inputElement = await page.$(
-			'figure[aria-label="Block: Image"] input[type=file]'
-		);
-
 		const filename = '1024x768_e2e_test_image_size.jpeg';
-		const filepath = './test/e2e/assets/' + filename;
-
-		await inputElement.setInputFiles( filepath );
```

Comment:
We already have dedicated e2e tests to test the uploading spec in `editor/blocks/image.test.js` so we don't have to use the UI to upload the file here.

This got me thinking though, why didn't we write this test in `editor/blocks/image.test.js`? Is there a reason why we created this test file separately?